rescript/src/TableclothMap.ml

```diff
@@ -36,6 +36,17 @@ let mapWithIndex t ~f = Belt.Map.mapWithKey t f
 
 let filter m ~f = Belt.Map.keep m (fun _ value -> f value)
 
+let filterMap m ~f =
```

Comment:
Rescript is no longer in this repo (it's in https://github.com/darklang/tablecloth-rescript)

Reply:
Would love a PR there.